Tests/TensorFlowTests/EpochsTests.swift

```diff
@@ -19,95 +19,156 @@ import XCTest
 var rng = ARC4RandomNumberGenerator(seed: [42])
 
 final class EpochsTests: XCTestCase {
-  func testBaseUse() {
-    // A mock item type that tracks if it was accessed or not
-    class AccessTracker {
+ 
+  // An element that keeps track of when it was first accessed.
+  class AccessTracker {
       var accessed: Bool = false
+  }
+  
+  // A struct keeping track of when its elements have been first accessed. We 
```

Comment:
No point ever in saying “a struct” in a comment.

Reply:
Addressed.